### PR TITLE
compose sample.

### PR DESCRIPTION
**Pull request by davehorner**

### CODE REVIEW

File: composeApp/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+//            implementation(libs.kstatemachine.coroutines)
+            val voyagerVersion = "1.1.0-beta02"
+
+            implementation("cafe.adriel.voyager:voyager-screenmodel:$voyagerVersion")
+//            implementation("cafe.adriel.voyager:voyager-navigator:$voyagerVersion")
+//            implementation("cafe.adriel.voyager:voyager-bottom-sheet-navigator:$voyagerVersion")
+//            implementation("cafe.adriel.voyager:voyager-tab-navigator:$voyagerVersion")
+//            implementation("cafe.adriel.voyager:voyager-transitions:$voyagerVersion")
+            //implementation("cafe.adriel.voyager:voyager-lifecycle-kmp:$voyagerVersion")
+//            implementation("media.kamel:kamel-image:0.9.5")
```
Comment (nsk90) [inline composeApp/build.gradle.kts:68]: could you clean dependency list? 
some of them seems to be useless here, like ktor for instance
I also don't know the purpose of kermit, and voyager libraries.

File: composeApp/src/commonMain/kotlin/org/example/project/XkcdClient.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package org.example.project
+
+import io.ktor.client.*
+import io.ktor.client.call.body
+import io.ktor.client.request.*
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import kotlinx.serialization.Serializable
+import io.ktor.serialization.kotlinx.json.*
+
+class XkcdClient {
```
Comment (nsk90) [inline composeApp/src/commonMain/kotlin/org/example/project/XkcdClient.kt:10]: can this class be removed?

File: iosApp/iosApp/iOSApp.swift [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import SwiftUI
```
Comment (nsk90) [inline iosApp/iosApp/iOSApp.swift:1]: looks that iOS part is not implemented, may be it's better to remove it?

File: composeApp/build.gradle.kts [hunk 4]
```diff
@@ -0,0 +1,124 @@
+import org.jetbrains.compose.desktop.application.dsl.TargetFormat
+import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+    alias(libs.plugins.kotlinMultiplatform)
+    alias(libs.plugins.androidApplication)
+    alias(libs.plugins.jetbrainsCompose)
+    alias(libs.plugins.compose.compiler)
+    kotlin("plugin.serialization") version "1.5.30"
```
Comment (nsk90) [inline composeApp/build.gradle.kts:10]: serialization is also seems to be redundant here

**Comment by davehorner** [comment]
Please feel free to remove the libraries; no ios, I was going to try that, but please feel free to make any changes you desire.  I'd keep kermit the logger but its your repo and I'd be happy to see how you change the code.

**Comment by nsk90** [comment]
Ok, thank you so much! Great job!)

**Pull request merged**